data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+				accept
+	on visit
+		dialog phrase "generic waypoint on visit"
+
+
+
+mission "Hai Reveal: Meet the Team: Sayari"
+	landing
+	name "Important Message"
+	description "Deliver an important message from the Hai to the Republic on <stopovers>. Return to <destination> when finished."
+	source "Hai-home"
+	stopover "Earth"
+	to offer
+		has "Hai Reveal: Meet the Team: Terry: done"
+	on offer
+		conversation
+			`Terry quickly takes a copy of your ship's system logs upon your return. "Thank you, Cpatain. I'll have these analyzed in no time." Terry runs back into the building, passing Sayari on her way out, carrying a small box.`
+			`	"Captain <last>. I know we are asking much of you, but I have a task of my own." She hands you the box, which is officially branded by the Hai government. Inside is an encrypted data chip and a metallic badge. There is a symbol minutely engraved into the front that must be the Hai government's emblem. Surrounding it in a circle is some sort of pattern, which reminds you of the codes found on identification cards and licenses in human space. On the other side, there is extensive writing in a script that looks to be Hai. From this, you presume that the badge is some sort of diplomatic ID.`
+			`	"Please deliver this message to the Republic. I trust that they will disseminate this message to your other governments. This badge should make it easier if they give you trouble."`
+			choice
+				`	"Consider it done."`
+					accept
+				`	"Various? We only have two governments."`
+			`	"Oh," she says with a smirk, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
+				accept
+	on stopover
+		conversation
+			`On approach to the spaceport on Earth, you request to speak to a diplomatic official. Given your renown from the Pug war, you're promise you a "speedy two to three business days" wait. But after showing the diplomatic ID that Sayari gave you, that quickly shrinks to "two hours."`
+			`	You meet with Jamal Rahman, the Chief Minister of Foreign Affairs and Xilin's boss. "I take it you've involved yourself with this 'Hai' situation," he says. You nod. "Well I apologize, but I do not have much time for small talk. What is it you need me for?" You hand him the encrypted message that Sayari gave you and explain where it's from. He thanks you, and leaves with the message.`
+			`	Some hours later, Rahman returns with the an encrypted data chip. "This is our response. Safe travels, Captain."`
+	on visit
+		dialog phrase "generic stopover on visit"
+	on complete
+		event "hr: exploration period" 45
+		conversation
+			`You meet Sayari at the secure port and hand her the response from the Republic. "Thank you, Captain <last>," she says. "Not only for this task, but for the other tasks you have completed for us. You've helped us gather much needed information to determine the path forward."`
```

Comment:
Technically all that information could be acquired without you - but your involvement meant that the team did not have to choose between trusting a Hai captain, or a random merchant captain, and could rely on someone well-vouched for.
```suggestion
			`You meet Sayari at the secure port and hand her the response from the Republic. "Thank you, Captain <last>," she says. "Not only for this task, but for the other tasks you have completed for us. You've helped with many tasks which might otherwise have been sources of tension without you."`
```